api_tests/src/wallets/bitcoind_wallet.ts

```diff
@@ -0,0 +1,88 @@
+import { BitcoinWallet } from "comit-sdk";
+import { BitcoinNodeConfig } from "../ledgers";
+import BitcoinRpcClient from "bitcoin-core";
+
+export class BitcoindWallet implements BitcoinWallet {
+    constructor(private readonly client: BitcoinRpcClient) {}
+
+    public static async newInstance(
+        network: string,
+        hdKey: string,
+        config: BitcoinNodeConfig
+    ): Promise<BitcoindWallet> {
+        const bitcoinRpcClient = new BitcoinRpcClient({
+            network,
+            host: "localhost",
+            port: config.rpcPort,
+            username: config.username,
+            password: config.password,
+            wallet: "miner",
```

Comment:
Is it required that we are using the `miner`s wallet here? From what I can see, we are not using this wallet here right?